css/at-rules/media.json

```diff
@@ -1330,7 +1338,8 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": false
+                "version_added": false,
+                "impl_url": "https://webkit.org/b/175497"
```

Comment:
I believe it's correct to add an impl_url even if the support is currently false?

I have an implementation in WebKit that's waiting in review and subsequent release processes but it can be tracked at that bug.